apps/shell/test/helpers.test.js

```diff
@@ -87,5 +89,23 @@ describe('Helper function definedHosts()', () => {
     expect(defaultHost).toEqual('pitzer.osc.edu');
     expect(helpers.definedHosts()['hosts']).toEqual(['pitzer.osc.edu', 'ruby.osc.edu']);
   })
+
+  test('when an unreadable yml file is present', () => {
+    const unreadableYmlPath = path.join(process.env.OOD_CLUSTERS, 'pitzer.yml');
+    fs.chmodSync(unreadableYmlPath, 0);
+    try {
+      let hosts = helpers.definedHosts();
+      let defaultHost = hosts['default'];
+      let allHosts = hosts['hosts'];
+      // owens.yml has default in it
+      expect(defaultHost).toEqual('owens.osc.edu');
+      expect(allHosts).toEqual(['owens.osc.edu', 'ruby.osc.edu']);
+      fs.chmodSync(unreadableYmlPath, 420);
```

Comment:
I'm more used to the octal notation `644` or `755`.  Can you change this in the other 420 too?

```suggestion
      fs.chmodSync(unreadableYmlPath, 0o644);
```